test/e2e/network/kube_proxy.go

```diff
@@ -186,27 +187,25 @@ var _ = SIGDescribe("Network", func() {
 		ginkgo.By("Checking /proc/net/nf_conntrack for the timeout")
 		// If test flakes occur here, then this check should be performed
 		// in a loop as there may be a race with the client connecting.
-		e2essh.IssueSSHCommandWithResult(
+		result, err := e2essh.IssueSSHCommandWithResult(
 			fmt.Sprintf("sudo cat /proc/net/nf_conntrack | grep 'dport=%v'",
 				testDaemonTCPPort),
 			framework.TestContext.Provider,
 			clientNodeInfo.node)
+		framework.ExpectNoError(err)
 
 		// Timeout in seconds is available as the fifth column from
 		// /proc/net/nf_conntrack.
-		result, err := e2essh.IssueSSHCommandWithResult(
+		re := regexp.MustCompile(
 			fmt.Sprintf(
-				"sudo cat /proc/net/nf_conntrack "+
-					"| grep 'CLOSE_WAIT.*dst=%v.*dport=%v' "+
-					"| tail -n 1"+
-					"| awk '{print $5}' ",
+				"^\\w+\\s+\\w+\\s+\\w+\\s+\\w+\\s+(\\d+)\\s+CLOSE_WAIT.*dst=%s.*dport=%d",
```

Comment:
would be cleaner with back-ticks to avoid double backslashes, and a `{4}` pattern instead of repeating `\w+\s+` 4x:

```go
`^(\w+\s+){4}(\d+)\s+CLOSE_WAIT.*dst=%s.*dport=%d`
```

Reply:
absolutely, thanks for the tip